src/word.c

```diff
@@ -0,0 +1,67 @@
+#include "dictionary.h"
+#include <stdio.h>
+#include <strings.h>
+#include <string.h>
+#include <assert.h>
+#include <stdlib.h>	
+
+int num_punctuation = 22;
+char* punctuation_array[] = {"+",",",":"," ",".","-","'","&","!","?",":",";","#","~","=","/","$","£","^","\n","_","<",">"};
```

Comment:
This should be put on multiple lines so we don't go so far over the generally accepted line limit of about 80 characters.

Reply:
done